src/components/button/Button.tsx

```diff
@@ -5,16 +5,18 @@ interface ButtonProps {
   children?: React.ReactNode;
   disabled?: boolean;
   onClick?: () => void;
+  className?: string;
 }
 
 const Button: React.FC<ButtonProps> = ({
   children,
   disabled = false,
   onClick = () => {},
+  className,
 }) => {
   return (
     <button
-      className={`${s.button} ${s.default}`}
+      className={`${s.button} ${s.default}  ${className}`}
```

Comment:
Kiedy nie podamy className w propsach to w naszym DOM pojawi sie klasa "undefind"